{
  "name": "test_delcontact",
  "status": "failed",
  "statusDetails": {
    "message": "AssertionError: assert '2' == 2",
    "trace": "self = <test_contact.TestContact object at 0x0000025F0EAE88D0>\n\n    def test_delcontact(self):\n        delpage = self.main.goto_contactlist().\\\n            search_contact().personal_infor().\\\n            person_setting().contactedit().\\\n            contactdelpage()\n        text = delpage.get_contact()\n>       assert text == 2\nE       AssertionError: assert '2' == 2\n\ntest_contact.py:64: AssertionError"
  },
  "attachments": [
    {
      "name": "log",
      "source": "6cd798c9-3bce-4bb3-be80-2e54abd1561d-attachment.txt",
      "type": "text/plain"
    },
    {
      "name": "stderr",
      "source": "dcaf6741-97cb-40b1-8846-855bdc8ca208-attachment.txt",
      "type": "text/plain"
    }
  ],
  "start": 1595938098650,
  "stop": 1595938125383,
  "uuid": "821a720f-0f47-4aad-889b-bef77635aad1",
  "historyId": "8e762da2be401352d7fdbd6f2e409651",
  "testCaseId": "af86a1443588099abc375036be956dd7",
  "fullName": "test_contact.TestContact#test_delcontact",
  "labels": [
    {
      "name": "suite",
      "value": "test_contact"
    },
    {
      "name": "subSuite",
      "value": "TestContact"
    },
    {
      "name": "host",
      "value": "X8O0ILYDG7L3A0P"
    },
    {
      "name": "thread",
      "value": "38156-MainThread"
    },
    {
      "name": "framework",
      "value": "pytest"
    },
    {
      "name": "language",
      "value": "cpython3"
    },
    {
      "name": "package",
      "value": "test_contact"
    }
  ]
}